### PR TITLE
feat(auth): Sign-in screen delete sign-out button

### PR DESCRIPTION
**Pull request by LeiChen5**
This PR removes the sign-out button from the sign-in screen to ensure correct behavior.

Previously, the sign-in screen included a sign-out button, which was used for testing the sign-in functionality. After successfully signing in, this button was immediately displayed. Now that we have an overview screen that users should be directed to after signing in, the sign-out button is no longer necessary.

Changes made:
-**SignIn.kt**: Removed the sign-out button, added the package declaration at the top of the file, and assigned test tags to the components.
-**MainActivity.kt**: Updated the import statement for SignInScreen to reflect the new package name.

Additionally, I have included tests to verify that all components are correctly displayed and changed our app's logo to a new one.

### CODE REVIEW

**Comment by LeiChen5** [comment]
The tests pass on my PC but fail in the CI environment. I'm currently working on a solution.

**Comment by LeiChen5** [comment]
The tests were passing on my PC but failing in the CI environment because the emulator configurations were different: the CI was using a small phone emulator, while I was using a medium phone emulator. After switching my local emulator to a small phone, I was able to reproduce the same error. I resolved it by using **withContext()** for displaying the Toast message.

The new code currently shows 0.0% line coverage, which is expected since the **SignIn.kt** file was pushed to the main branch during Sprint 1. At that time, we weren't aware that both code and tests should be included in the same PR. I’ve now added the tests to ensure we have overall coverage in SonarCloud.

File: app/src/androidTest/java/com/android/brewr/ui/authentication/SignInTest.kt [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+@RunWith(AndroidJUnit4::class)
+class SignInScreenTest {
+
+  @get:Rule val composeTestRule = createAndroidComposeRule<MainActivity>()
+
+  private lateinit var navigationActions: NavigationActions
+
+  @Before
+  fun setup() {
+    navigationActions = mock()
```
Comment (yixiaoi) [inline app/src/androidTest/java/com/android/brewr/ui/authentication/SignInTest.kt:26]: i don't think the navigationActions is used anymore, maybe you could delete it.

**Review by yixiaoi — COMMENTED** [review]
the code is generally good, there is one part to delete

**Review by yixiaoi — COMMENTED** [review]
log in feature works well, there is no more log out page. Maybe it is time to change our logo to the new voted one.

**Review by yixiaoi — COMMENTED** [review]
Code with new logo is good.

**Review by yixiaoi — APPROVED** [review]
This code has successfully implemented the sign in screen with new logo and removed the sign out button.

**Pull request merged**